### PR TITLE
Back out "Revert D16469619: Add Virtual Memory and CPU percentage computation to AIBench"

### PR DESCRIPTION
**Pull request by geof90**
Differential Revision: D16654854

### CODE REVIEW

File: binaries/benchmark_helper.cc [hunk 1]
```diff
@@ -376,6 +377,35 @@ void writeOutput(
...
+    const std::string& unit,
+    const int value) {
+  LOG(INFO) << caffe2::NetObserverReporterPrint::IDENTIFIER << "{"
+            << "\"type\": \"" << type << "\", "
+            << "\"metric\": \"" << metric << "\", "
+            << "\"unit\": \"" << unit << "\", "
+            << "\"value\": " << c10::to_string(value) << "}\n";
+}
+
+long getVirtualMemoryIfOptionEnabled(bool FLAGS_measure_memory) {
```
Comment (peterjc123) [inline binaries/benchmark_helper.cc]: This function doesn't have an implementation for Windows. Would you please add them into your code?
There are two functions that may suit your needs:
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/heapwalk?view=vs-2019
https://docs.microsoft.com/zh-cn/windows/win32/psapi/collecting-memory-usage-information-for-a-process

Comment (geof90) [reply binaries/benchmark_helper.cc]: Thanks for the pointer, will work on it!

**Pull request closed without merging**